src/libraries/Native/Unix/System.Security.Cryptography.Native/openssl.c

```diff
@@ -1022,6 +1022,7 @@ int32_t CryptoNative_PushX509StackField(STACK_OF(X509) * stack, X509* x509)
     return sk_X509_push(stack, x509);
 }
 
+#ifndef USE_ANDROID_CRYPTO
```

Comment:
Kinda makes me wonder if we should "follow Apple" and use an `Apple` prefix in the new function names, a'la `System.Security.Cryptography.Native.Apple`? https://github.com/dotnet/runtime/blob/88c46f875cf8189ca0e666d534847e801c72272e/src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_random.h#L14

Not sure what that means/looks like on the managed side, though.